manifests/e/Ergosoft/ErgoFAKT_V5/4.9.2.5294/Ergosoft.ErgoFAKT_V5.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Ergosoft.ErgoFAKT_V5
+PackageVersion: 4.9.2.5294

```

Comment:
```suggestion
PackageVersion: 4.9.025294
```

Reply:
4.9.2.5294 does not reflect the msi-installer version, but ist the correct version of the product, so this is intended.

Reply:
Unfortunately, the version number must match Control Panel.

If you think this is incorrect then please update your software with the correct version and update `InstallerUrl` and `InstallerSha256`.

Reply:
Last Message: "The package manager bot determined that the metadata was not compliant."

Seems that changing the PackageVersion has broken something in the pipeline (maybe a new directory structure is required )?

Reply:
New "Manifest-Validation-Error":
The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\e\Ergosoft\ErgoFAKT_V5\4.9.2.5294', expected 'manifests\e\Ergosoft\ErgoFAKT_V5\4.9.025294'

How to resolve this "manifest file path" conflict, introduced by the change of the version number ?

Should I simply close this Pull Request and run "wingetcreate submit" again (with the corrected PackageVersion) ?

Reply:
You can use e.g. Github Desktop to rename it.